config/vufind/config.ini

```diff
@@ -350,6 +350,22 @@ title_level_holds_mode = "disabled"
 ; override ILS-provided default pickup locations throughout the system.
 set_home_library = true
 
+; Allow the user to "subscribe" to items of search history in order to receive
+; email notifications of new items.
```

Comment:
I'd change "new items" to "new search results".

Reply:
Agreed; I've tweaked the language.